src/main/java/seedu/address/model/person/InterviewStatus.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.model.person;
+
+public class InterviewStatus {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "Interview Status should be Pending, Interviewing or Not Interviewed";
+    public static final String PENDING_STATUS = "Pending";
+    public static final String INTERVIEWING_STATUS = "Interviewing";
+    public static final String NOT_INTERVIEWED_STATUS = "Not Interviewed";
+
+    private String interviewStatus;
+
+    /**
+     * Constructor for Interview Status.
+     *
+     * @param interviewStatusType the string value of the status.
+     */
+    public InterviewStatus(String interviewStatusType) {
+        String input = interviewStatusType.toLowerCase();
+        input = input.replaceAll(" ", "");
+
+        if (input.equals(PENDING_STATUS.toLowerCase())) {
+            this.interviewStatus = PENDING_STATUS;
+        } else if (input.equals(INTERVIEWING_STATUS.toLowerCase())) {
+            this.interviewStatus = INTERVIEWING_STATUS;
+        } else if (input.equals(NOT_INTERVIEWED_STATUS
+                .replaceAll(" ", "").toLowerCase())) {
+            this.interviewStatus = NOT_INTERVIEWED_STATUS;
```

Comment:
Perhaps this could be changed as well. 🤔

Reply:
@tiewweijian will you be updating this now or in future PRs? If not I will merge this in!